Adds two new pages for sending notifications with messages and NOR co… (#34)
labopm/competitive-hiring-pilot: pages/phases/phone-assessment-interviews/notifications.md

---
layout: article
permalink: /hiring-phases/phone-assessment-interviews/notifications/
section: hiring-phases
title: Sending Notifications Post Phone Assessment Interview
sidenav: hiring-phases
phase: phone-assessment-interviews
sub-phase-order: 040

> intro: It is appropriate to send notifications to all applicants at the end of each phase. Use the following NOR codes and notification messages to communicate with applicants.

Reviewer: ```suggestion
intro: As a best practice, you should send notifications to all applicants at the end of each phase. Use these NOR codes and notification messages to communicate with applicants.
```
